packages/graphql/src/schema/query.ts

```diff
@@ -361,7 +361,7 @@ export const Query = builder.queryType({
           id: t.arg.string({ required: true }),
         },
         resolve(_, args, ctx) {
-          return SessionService.getSessionEvaluation(args, ctx)
+          return SessionService.getSessionEvaluation(args, ctx) as any
```

Comment:
remove as any once all type issues in sessions service are fixed

Reply:
also holds for all of the other as any in query.ts or mutation.ts, once we have clean typings everywhere, we should be able to remove all of them.